### ISSUE TITLE
Reorganizing password store content

### ISSUE THREAD

**Issue by dominikschulz** [gopasspw/gopass#1866]
Assume one has a large password store that has grown over time, with no consistent folder structure.

What would be that easiest way to reorganize these?

A bash script would work, but I think that's bit too advanced for many users.
Maybe we could dump all secret names into a text file and launch an editor that allows easily reorganizing them?

**Comment by innovate-invent** [comment]
cant you just reorganise `~/.password-store`? and manually commit it?

**Comment by dominikschulz** [comment]
Yes, sure. I was wondering if we could provide a more user friendly interface to that.
But if you're proficient with the command line and git you can definitely do that.
